packages/core/index.d.ts

```diff
@@ -286,6 +288,22 @@ declare module '@rjsf/core' {
 
     export module utils {
 
+        type UseEmptyValueOnChangeProps = {
```

Comment:
I'm definitely not super strong at typescript. If someone has a suggestion how to clean this up, it would be much appreciated.